contracts/LoanCore.sol

```diff
@@ -233,4 +234,26 @@ contract LoanCore is ILoanCore, AccessControl, Pausable {
     function unpause() external onlyRole(DEFAULT_ADMIN_ROLE) {
         _unpause();
     }
+
+    /**
+     * @inheritdoc ICallDelegator
+     */
+    function canCallOn(address caller, address vault) external view override returns (bool) {
+        // if the collateral is not currently being used in a loan, disallow
```

Comment:
Not sure I get this, why disallow? Shouldn't they have _more_ freedom to call fns if the collateral is not being used?